test/scripts/dynamic-fee-teal-test.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+date '+dynamic-fee-teal-test start %Y%m%d_%H%M%S'
+
+set -e
+set -x
+set -o pipefail
+export GOPATH=$(go env GOPATH)
+
+TEMPDIR=$(mktemp -d)
+trap "rm -rf $TEMPDIR" 0
+
+NETDIR=${TEMPDIR}/net
+
+if [ ! -z $BINDIR ]; then
+    export PATH=${BINDIR}:${PATH}
+fi
+
+goal network create -r ${NETDIR} -n dynamic-fee-teal-test -t ${GOPATH}/src/github.com/algorand/go-algorand/test/testdata/nettemplates/TwoNodes50EachFuture.json
```

Comment:
might not use TwoNodes50EachFuture anymore, TwoNodes50Each also works

Reply:
updated, thanks